docs/attributes-dictionary/http.md

```diff
@@ -0,0 +1,53 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: HTTP
```

Comment:
We don't need a `linkTitle` unless it's a shorter version of the `title`:

```suggestion
```

Reply:
@chalin 
Do we still need the `<!--- Hugo front matter used to generate the website version of this page:` line then? Or should I remove the entire `Hugo` comment here?